conf/radiusd/packetfence.example

```diff
@@ -257,9 +260,8 @@ accounting {
 		&control:PacketFence-RPC-User = ${rpc_user}
 		&control:PacketFence-RPC-Pass = ${rpc_pass}
 		&control:PacketFence-RPC-Proto = ${rpc_proto}
+		&request:Calling-Station-Id = "%{User-Name}"
```

Comment:
In fact it will set the Calling-Station-Id attribute to the user-name if only Calling-Station-Id is undef.
Btw you are right, i will change the policy to something like that:
rewrite_user_name {
         if (&User-Name && (&User-Name =~ /^${policy.mac-addr-regexp}$/i)) {
                 update request {
                         &User-Name := "%{tolower:%{1}:%{2}:%{3}:%{4}:%{5}:%{6}}"
                         &Calling-Station-Id =  "%{tolower:%{1}:%{2}:%{3}:%{4}:%{5}:%{6}}"
                 }
                 updated
         }
         else {
                 noop
         }
 }
And remove &request:Calling-Station-Id = "%{User-Name}"